S2VX.Web/Pages/Counter.razor

```diff
@@ -0,0 +1,16 @@
+﻿@page "/counter"
+
+<h1>Counter</h1>
+
+<p>Current count: @currentCount</p>
+
+<button class="btn btn-primary" @onclick="IncrementCount">Click me</button>
+
+@code {
+    private int currentCount = 0;
+
+    private void IncrementCount()
+    {
+        currentCount++;
+    }
+}
```

Comment:
Remove this and other unused pages page

Reply:
Ok, I removed Counter.razor and FetchData.razor in the latest commit